htmd/molecule/molecule.py

```diff
@@ -1279,6 +1232,15 @@ def write(self, filename, sel=None, type=None, **kwargs):
             raise IOError('Molecule cannot write files with "{}" extension yet. If you need such support please notify '
                           'us on the github htmd issue tracker.'.format(ext))
 
+    def _emptyTopo(self, numAtoms):
+        for field in Molecule._atom_fields:
```

Comment:
shouldn't it be `Molecule._topo_fields`?

Reply:
No, we don't initialize the ['bonds', 'bondtype', 'angles', 'dihedrals', 'impropers', 'crystalinfo'] fields. Just the ones with atom information. The others can be empty.